db/schema.rb

```diff
@@ -204,12 +204,13 @@
     t.boolean "psychiatric_liaison"
     t.string "psychiatric_liaison_reason_not"
     t.boolean "next_hearing"
+    t.integer "travel_time"
```

Comment:
I think something has gone a bit wrong with your rebase - we don't want to be reverting this change, and especially not by simply deleting the migration files, as that will break every environment where those migrations have already been run.